src/test/java/FooterTest.java

```diff
@@ -0,0 +1,94 @@
+import components.Footer;
+import enums.Attribute;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import utils.WebDriverUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FooterTest {
+    private static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
+    private static final String TESTING_BOARD_URL = "https://softwaretestingboard.com/";
+    private final By footerLink = By.cssSelector(".footer.content ul.footer.links a");
+    private final By footerName = By.cssSelector(".footer.content ul.footer.links a");
+    private static final String WHITE_COLOR = "#ffffff";
+    private static final String BLUE_COLOR = "#1979c3";
+    private WebDriver driver;
+    private Footer footer;
+    private List<WebElement> footerCategoriesList;
```

Comment:
Why is it defined as a class field? It's only used ones in footerNavigation_FirstLevelCategories_CorrectlyDisplayed(), so should be defined there.